lib/credit_card_form.dart

```diff
@@ -281,7 +281,7 @@ class _CreditCardFormState extends State<CreditCardForm> {
                   validator: widget.cardNumberValidator ??
                       (String? value) {
                         // Validate less that 13 digits +3 white spaces
-                        if (value!.isEmpty || value.length < 16) {
+                        if (value!.isEmpty || value.length < 19) {
```

Comment:
These changes should be based on the brand, as most of the brands support only 16 digits.